Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -50,6 +51,8 @@ private function resolveInterpreterExceptionMessage(InterpreterException $except
             return "Row is expected to contain {$exception->getExpectedSize()}, found to have {$exception->getFoundSize()} fields instead.";
         } elseif ($exception instanceof InvalidRowException) {
             return $this->buildInvalidRowExceptionMessage($exception);
+        } elseif ($exception instanceof BooleanFormatException) {
```

Comment:
Not sure it makes sense to create a class just to remove it again straight after. Maybe it's better to refactor first and then use?